management/appointments/appointmentsController.js

```diff
@@ -139,6 +139,24 @@ define('appointmentsController', [], function() {
 			$scope.appointment = null;
 		}
 
+		$scope.initializeCancelConfirmationModal = function() {
+			WDN.initializePlugin('modal', [ function() {
+				require(['jquery'], function($) {
```

Comment:
The UNL Colorbox example does it like this:
```
    WDN.initializePlugin('modal', [function() {
        var $ = require('jquery');
        $('.group1').colorbox({rel: 'group1'});
    }]);
```
It's a little tough to read with like 4 functions in a row here.

Also, can you guys use arrow functions? That would be fantastic here to clean it up a little.

Reply:
Good note on the UNL Colorbox example--it works properly and is much cleaner. Changed to that. Additionally, I will start using arrow functions over regular `function` declarations.